unittest/buildtool/bom_command_test.py

```diff
@@ -82,7 +82,6 @@ def test_default_bom_options(self):
     registry = {}
     buildtool.bom_commands.register_commands(registry, self.subparsers, {})
     self.assertTrue('build_bom' in registry)
-    self.assertTrue('publish_bom' in registry)
```

Comment:
I'd love to hear the reasoning behind this change.

Reply:
See: https://github.com/spinnaker/buildtool/commit/af0f7d96112a49e07a841017322d5620352b6b58

The command is in [BomPublisher.kt](https://github.com/spinnaker/spinrel/blob/master/jenkins-cli/src/main/kotlin/io/spinnaker/spinrel/cli/jenkins/BomPublisher.kt) and then there are various PR's leading up to that, eg: https://github.com/spinnaker/spinrel/pull/10/files

I'm not sure why `spinrel` was created. It looks like some tasks have moved from python `buildtool` to kotlin `spinrel` but I'm not sure what functionality changes or why.
[spinrel first commit](https://github.com/spinnaker/spinrel/commit/b8b2bb0e4ed7ef5f81eb2408f8347f38fb1267e9).

This PR assumes the fastest way to a 1.27.0 release is via re-using what already exists, with some minimal changes. This looks promising..

That said I haven't made it through the whole process yet (including publishing BOM) and unsure where we will end up, eg:
1. use `buildtool` exclusively (maybe reverting `publish_bom` removal?). Maybe delete no longer necessary commands if keep longer term?
2. use `buildtool` and `spinrel`
3. do something else (like the POC Go)

Reply:
Also [`publish_bom`](https://github.com/spinnaker/buildtool/commit/af0f7d96112a49e07a841017322d5620352b6b58) command was written to shell out to a locally running Halyard (via `HalRunner`) to perform the publish to GCS.
Halyard in turn uses the Google SDK to publish to GCS.